packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/legacy/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import 'angular-translate';
+
+import component from './component';
+import './styles.less';
+
+const moduleName = 'ovhManagerDedicatedCloudLegacy';
+
+angular
+  .module(moduleName, [])
+  .component('dedicatedCloudLegacy', component)
```

Comment:
Component name seems too generic in my opinion.

Is the `components` folder required?  I mean you could probably only keep the following architecture:

- (...)/dedicatedCloud/datacenter/backup/legacy/